internal/staging/start_testrunner.go

```diff
@@ -63,38 +64,31 @@ func (c *controller) startTesting(queue *s2hv1.Queue) error {
 	}
 
 	// get result from tests (polling check)
-	finished := true
 	testCondition := v1.ConditionTrue
 	message := "queue testing succeeded"
 	for _, testRunner := range testRunners {
 		testRunnerName := testRunner.GetName()
 		testResult, err := c.getTestResult(queue, testRunner)
-		if err != nil {
-			return err
-		}
 
```

Comment:
should we update `getTestResult` func https://github.com/agoda-com/samsahai/blob/ba1d8924c9e358f639277e85094cdb0d1788a5cd/internal/staging/start_testrunner.go#L192

to check `isBuildFinished` flag
if `!isBuildFinished` then return err and we can do the retry by checking the error
in a case like calling API fails due to a network issue

Reply:
added retry condition kub

Reply:
I've added the trigger test timeout Kub. 

so right now have 
* test trigger timeout, will be looping until reach timeout
* and also Retry getting test runner result.

WDYT?